arch-design/dashboard/dashboard_requirements_analysis_k_release.md

```diff
@@ -0,0 +1,334 @@
+# SODA Dashboard requirements for K release
+
+  
+
+**Author(s)**: [Anvith KS](https://github.com/anvithks)
+
+Major Version Updates
+
+|**Date** | **Version** | **Description** |
+|---------|-------------|-----------------|
+| 13th September 2021  | v0.1  | This document lists the requirements and changes needed for SODA Dashboard for K release. |
+
+
+
+## Goals
+- Delfin UI updates
+    - Update masking views UI
+        - Add Support for Volume Groups API
+        - Add support for Port Groups API
+        - Add support for Storage Host Groups API
+        
+- Bug fixes and improvements
+
+
+## Motivation and background
+
+As part of the quarterly release of SODA Delfin has enhanced the masking views with additional APIs. The same must be supported from the SODA Dashboard to allow users to experience these changes. Additionally any bug fixes and improvements will be covered as part of this release
+
+## Glossary
+
+**Masking view**
+---
+The masking view identifies the host group (set of hosts), the port group (set of array ports), and the storage group (set of volumes) in a combination which are associated. Masking view represent the LUN masking and mapping information. A masking view will have only one host group, storage group and port group
```

Comment:
one host or hostgroup and one volume or volume group and optionally one port or portgroup